Corzbank.Data/Enums/CardTypeEnum.cs

```diff
@@ -0,0 +1,13 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Text;
+
+namespace Corzbank.Data.Enums
+{
+    public enum CardTypeEnum
+    {
+        Credit,
```

Comment:
assign only first value to 1. There is a case where enum will have a default state that is equal to 0. Refactor in all places